tests/ci/codebuild/android/run_android_fips_static.yml

```diff
@@ -0,0 +1,14 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - chmod +x ./tests/ci/android/AWSLCAndroidTestRunner/gradlew
+      - cd tests/ci/
+      - python3 -m venv .env && . .env/bin/activate && pip install -r requirements.txt
+      - >
+        ./kickoff_devicefarm_job.sh
```

Comment:
Do these need to have a `\` before the next line? Not sure if codebuild auto-handles that as part of the `- >`

Reply:
Yeah I remember wrestling with Codebuild when working with these yml files.
Apparently the yml files still don't recognize the subsequent line, when I put `\` at the end. Using `- >` was the way to tell yml to define a multi-line bash command in the script file.